RTN-004.tex

```diff
@@ -323,8 +358,8 @@ \section{Resources for DP0 Delegates}\label{sec:res}
 
 \subsection{Policies for RSP Accounts}\label{ssec:res_pol}
 
-For the duration of DP0, the RSP will be installed in the Interim Data Facility (IDF) deployed on the Google Cloud Platform.
-More information about the IDF and its installation will be available elsewhere.
+For the duration of DP0, the RSP will be installed in the Interim Data Facility (IDF), which will be deployed on the Google Cloud Platform\footnote{\url{https://www.lsst.org/news/rubin-observatory-will-partner-google-cloud-host-interim-data-facility}}.
```

Comment:
DP0, an instance of the RSP will be running at the Interim Data Facility (IDF), which ....

Reply:
fixed